README.md

```diff
@@ -72,16 +73,16 @@ For **ARM Devices** (Such as a Raspberry Pi) things might be a bit different. If
 _NOTE: Keep in mind that you might need to try multiple recovery images before you find a working one._ 
 
 # Copyrights and Licenses #
+
 Retrospect is licensed under a _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_. See also: [LICENSE.md](./LICENSE.md)
 
-The official add-on packages that are currently available within the official Retrospect Repository may not be distributed via other channels than the official Retrospect Repository. Only the official Retrospect Respository (respository.retrospect) package itself may be distributed and/or included within other Kodi (super) repositories.
+The official add-on package currently available within the official Kodi add-on repository may not be distributed via other channels than the official Kodi add-on repository. 
 
 ### Disclaimer ###
-Retrospect or Rieter.net are not connected to or in any other way affiliated with Kodi, Team Kodi, or the Kodi Foundation. Furthermore, any software, addons, or products offered by Retrospect or Rieter.net will receive no support in official Kodi channels, including the Kodi forums and various social networks.
+Retrospect or Rieter.net are not connected to or in any other way affiliated with Kodi, Team Kodi or the Kodi Foundation. Furthermore, any software, addons, or products offered by Retrospect or Rieter.net will receive no support in official Kodi channels, including the Kodi forums and various social networks.
 
```

Comment:
I think this part needs to be edited, since there is a Retrospect forum in forum.kodi.tv and Retrospect is in the official Kodi add-on repository. Maybe remove the last sentence entirely? Or "...will only receive support in its forum on forum.kodi.tv and Github repository."?